packages/indy-sdk/src/IndySdkModuleConfig.ts

```diff
@@ -0,0 +1,45 @@
+import type * as IndySdk from 'indy-sdk'
```

Comment:
Any reason this is imported as `IndySDK` as opposed to `Indy` in other parts of the code?

Reply:
I've tried to rename as much as possible to IndySDK, as Indy is a project, and IndySdk a specific implementation. Just to make it more clear it's the interface for the IndySdk